ArduPlane/servos.cpp

```diff
@@ -727,9 +727,35 @@ void Plane::servos_twin_engine_mix(void)
     }
 }
 
-
 /*
-  Set the flight control servos based on the current calculated values
+  Set throttle,attitude(in Attitude.cpp), and tilt servos for forced flare by RCx_OPTION switch for landing in FW mode
+  For Fixed Wind modes with manual throttle control only. Forces tilts up and throttle to THR_MIN.
+  Throttle stick must be low. This allows non-momentary switch to be used and quick bailouts
+  for go-arounds. Also helps prevent propstrike after landing with switch release on ground.
+*/
+void Plane::force_flare(void)
+{
+    if (!quadplane.in_transition() && !control_mode->is_vtol_mode() && !auto_throttle_mode && plane.channel_throttle->norm_input() < -0.95 && plane.flare_switch_active) {
+        int32_t tilt = -SERVO_MAX;
+        if (quadplane.tilt.tilt_type == QuadPlane::TILT_TYPE_BICOPTER) {
+            tilt = 0;
+        }
+        if (quadplane.is_tailsitter()) {
+            tilt = SERVO_MAX;
+        }
+        SRV_Channels::set_output_scaled(SRV_Channel::k_motor_tilt, tilt);
+        SRV_Channels::set_output_scaled(SRV_Channel::k_tiltMotorLeft, tilt);
+        SRV_Channels::set_output_scaled(SRV_Channel::k_tiltMotorRight, tilt);
+        float throttle_min = aparm.throttle_min.get(); //allows ICE to run if used
+        if (arming.is_armed()) {  //prevent running motors if unarmed
+            SRV_Channels::set_output_scaled(SRV_Channel::k_throttle, throttle_min);
```

Comment:
why isn't this using AP_LANDING somehow?

Reply:
Tridge wants to eventually add it to autoland, but thats a bit ambitious for me....I just wanted a belly land option for tailsitters and tiltrotors and Tridge suggested I expand it a bit....future enhancement to add to final flare in autolanding